### PR TITLE
Update jiramicrosoft-tutorial.md

### PR DESCRIPTION
**Pull request by SagarQA1695**
Hi Dhivya

Updated jiramicrosoft-tutorial.

Please check and approve the PR.

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@dhivyagana

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by dhivyagana — APPROVED** [review]

**Comment by dhivyagana** [comment]
#sign-off

File: docs/identity/saas-apps/jiramicrosoft-tutorial.md [hunk 1]
```diff
@@ -241,30 +249,35 @@ In this section, you'll enable B.Simon to use single sign-on by granting access
 	
 	d. In **Login Button Description** type the description of button your organization wants the users to see on login screen.
 
-	e. In **SAML User ID Locations** select either **User ID is in the NameIdentifier element of the Subject statement** or **User ID is in an Attribute element**.  This ID has to be the JIRA user ID. If the user ID is not matched, then system will not allow users to sign in.
+	e.  In **Default Group** Select your organization Default Group to assign to new users (Default groups facilitate organized access rights to new user account).
+
+	f. In **SAML User ID Locations** select either **User ID is in the NameIdentifier element of the Subject statement** or **User ID is in an Attribute element**.  This ID has to be the JIRA user ID. If the user ID is not matched, then system will not allow users to sign in.
```
Comment (Court72) [inline docs/identity/saas-apps/jiramicrosoft-tutorial.md:254]:
```suggestion
	f. In **SAML User ID Locations** select either **User ID is in the NameIdentifier element of the Subject statement** or **User ID is in an Attribute element**.  This ID has to be the JIRA user ID. If the user ID is not matched, then the system doesn't allow users to sign in.
```

**Comment by Court72** [comment]
Thanks, @dhivyagana. Note that the new screenshot linked to on line 239 needs to be added to the entra-docs repo before we can merge. Can you or @SagarQA1695 add the image?

#hold-off

File: docs/identity/saas-apps/jiramicrosoft-tutorial.md [hunk 2]
```diff
@@ -241,30 +249,35 @@ In this section, you'll enable B.Simon to use single sign-on by granting access
...
 
 	> [!Note]
 	> Default SAML User ID location is Name Identifier. You can change this to an attribute option and enter the appropriate attribute name.
 
-	f. If you select **User ID is in an Attribute element** option, then in **Attribute name** textbox type the name of the attribute where User ID is expected.
+	g. If you select **User ID is in an Attribute element** option, then in **Attribute name** textbox type the name of the attribute where User ID is expected.
+
+	h.  In **Auto-create User** feature (JIT User Provisioning): It automates user account creation in authorized web applications, without the need for manual provisioning. This reduces administrative workload and increases productivity. Because JIT relies on AzureAD’s login response enter the SAML-response attribute values, which includes the user's email address, last name and first name.
```
[the remark this answers is not in the document]
Comment (SagarQA1695) [reply docs/identity/saas-apps/jiramicrosoft-tutorial.md]: Changes Done.

**Comment by SagarQA1695** [comment]
Hi @Court72 

We've resolved all your suggested comments, please check and change the PR status.

Thanks
Sagar Patil

**Comment by Court72** [comment]
Thanks, @SagarQA1695. The new screenshot that's linked to on line 239 still needs to be added to the entra-docs repo before we can merge. Can you or @dhivyagana add the image to the repo?

**Comment by dhivyagana** [comment]
> Thanks, @SagarQA1695. The new screenshot that's linked to on line 239 still needs to be added to the entra-docs repo before we can merge. Can you or @dhivyagana add the image to the repo?

@Court72 : Referred file is there in the repo but it not added to the file change (pull request). Can you help us on how to add it to the pull request?
https://github.com/SagarQA1695/azure-docs/blob/main/articles/active-directory/saas-apps/media/jiramicrosoft-tutorial/SSO_Plugin_Configuration_Page.png

**Comment by Court72** [comment]
@dhivyagana - Thanks for reaching out. For information about creating a screenshot, see [Create a screenshot for documentation](https://review.learn.microsoft.com/help/contribute/contribute-how-to-create-screenshot?branch=main). The [Requirements for naming a screenshot file](https://review.learn.microsoft.com/help/contribute/contribute-how-to-create-screenshot?branch=main#requirements-for-naming-a-screenshot-file) section has guidance on where to save an image in the repo. If you have any more questions, you can post them on the [Learn Platform Support channel on Teams.](https://teams.microsoft.com/l/channel/19%3A7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)

**Review by SagarQA1695 — COMMENTED** [review]
Modified image path

**Comment by SagarQA1695** [comment]
Hi @Court72 , We have update the screen-shot path, Can you please reverify ?

Incase of any please let us know.

Thanks 
Sagar Patil

**Comment by v-ymundhe** [comment]
Hi @ShannonLeavitt @Court72 @dhivyagana 
We have raised new PR as we faced uploaded image not showing in repo for current PR. Please review below new PR 

New PR: https://github.com/MicrosoftDocs/entra-docs/pull/1218

Thanks 
Yogesh M

**Comment by omondiatieno** [comment]
@ShannonLeavitt, can we proceed to close this PR since the work has been done in the other PR you've merged?

**Pull request closed without merging**